src/features/targeting/types.js

```diff
@@ -1,35 +1,8 @@
 // @flow
-import type { SetEligibleMessageHeaders, TargetingApi } from './targeting-api';
 import { MESSAGE_TYPES } from './constants';
 
 export type MessageType = $Values<typeof MESSAGE_TYPES>;
 
 export type MessageID = number;
 
 export type MessageName = string;
-
-export type MessageHeader = {|
```

Comment:
I removed those fields as I figured out I don't need them in targeting context. Those fields stay with in app messenger.